dockers/docker-orchagent/Dockerfile.j2

```diff
@@ -5,7 +5,7 @@ ENV DEBIAN_FRONTEND=noninteractive
 
 RUN apt-get update
 
-RUN apt-get install -f -y ifupdown bridge-utils libdbus-1-3 libdaemon0 libjansson4
+RUN apt-get install -f -y ifupdown libdbus-1-3 libdaemon0 libjansson4
```

Comment:
i wonder if we still want to have a copy of bridge-utils inside the swss docker. i'm open to this.